reviewing/plugin-commands-listing/src/list.ts

```diff
@@ -175,6 +177,7 @@ export async function render (
     onlyProjects: opts.onlyProjects,
     reportAs: (opts.parseable ? 'parseable' : (opts.json ? 'json' : 'tree')) as ('parseable' | 'json' | 'tree'),
     showExtraneous: false,
+    modulesDir: opts.modulesDir,
   }
```

Comment:
virtualStoreDir should also be passed, shouldn't it?

Reply:
We can [get `virtualStoreDir` from `node_modules/.modules.yaml`](https://github.com/pnpm/pnpm/blob/main/reviewing/dependencies-hierarchy/src/buildDependenciesHierarchy.ts#L51), So i think no need to pass `virtualStoreDir` here.

Reply:
but why if virtualStoreDir is already in `opts`?